src/main/java/janggi/domain/piece/PieceType.java

```diff
@@ -0,0 +1,39 @@
+package janggi.domain.piece;
+
+import janggi.domain.Team;
+import java.util.Arrays;
+
+public enum PieceType {
+    General("한", "초"),
```

Comment:
```suggestion
    GENERAL("한", "초"),
```
enum의 요소는 대문자로 쓰는게 일반적입니다ㅎㅎ

Reply:
급하게 구현하다 보니 놓친 요소였던 것 같습니다. 해당 요소 수정 완료하였습니다!

```java
public enum PieceType {
    GENERAL("한", "초"),
    CHARIOT("차", "차"),
    ELEPHANT("상", "상"),
    GUARD("사", "사"),
    HORSE("마", "마"),
    SOLDIER("병", "졸"),
    CANNON("포", "포"),
    ;
...
```